Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
Indeed, given the [current documentation of Symfony about conventions](http://symfony.com/doc/current/contributing/code/conventions.html) misses some of these aspects, I'm working now in a PR for that.